roles/elk/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: install oauth2_proxy
+  command: '/usr/local/bin/bash -c "export GOROOT=/usr/local/go GOPATH=/usr/local; go get github.com/bitly/oauth2_proxy"'
```

Comment:
Any reason to use the command module instead of shell for this?

Actually could use the command module if GOROOT and GOPATH are set in the environment option. I'm not sure how that will work with proxy_env though.